### ISSUE TITLE
How can I get extension on WEB?

### ISSUE THREAD

**Issue by michaldev** [miguelpruivo/flutter_file_picker#491]
I try like this:
```
  var result = await FilePicker.platform.pickFiles(type: FileType.custom, allowedExtensions: ['jpg', 'pdf'], allowMultiple: false);
  print(result.files.single.extension);
```

But this work only on mobile devices.

**Comment by miguelpruivo** [comment]
@michaldev it should work on web as well. Which browser are you trying on?

**Comment by michaldev** [comment]
Chrome. I got null.

**Comment by miguelpruivo** [comment]
It shouldn't actually. Have you tried with a different one?

**Comment by sultanmyrza** [comment]
have same issue path is null on flutter web
I created more detailed issue for others to follow 
https://github.com/miguelpruivo/flutter_file_picker/issues/495

**Comment by miguelpruivo** [comment]
@sultanmyrza #495 is closed since it’s working as intended. You must use `bytes` property on the web since absolute paths aren’t supported by browsers.

**Comment by sultanmyrza** [comment]
@miguelpruivo  I'm trying to do something like this but can't make it work on web
```
    FilePickerResult result = await FilePicker.platform.pickFiles();

    if (result != null) {
      var bytes = result.files.single.bytes;
      var fileName = result.files.single.name;

      // dart:html 
      var file = File(bytes, fileName);
      // dart: io
      var file = await File(fileName).writeAsBytes(bytes);

      // TODO: upload to firebase storage
      UploadTask task = uploadFile(file, fileName);
```

**Comment by miguelpruivo** [comment]
Closing as this should be fixed on the latest version (2.0.13).

Feel free to reopen if you still experience any problem related to it.

Thank you!
